yb-voyager/cmd/exportData.go

```diff
@@ -1669,6 +1578,125 @@ func startFallBackSetupIfRequired() {
 	}
 }
 
+// ================================ Export Data table list filtering ================================
```

Comment:
just broken down the original implementation into smaller functions for simplicity